inference-engine/src/transformations/src/transformations/op_conversions/convert_ctc_greedy_decoder_v6_to_v1.cpp

```diff
@@ -0,0 +1,69 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "itt.hpp"
+
+#include "transformations/op_conversions/convert_ctc_greedy_decoder_v6_to_v1.hpp"
+
+#include <ngraph/opsets/opset4.hpp>
+#include <ngraph/opsets/opset6.hpp>
+#include <ngraph/rt_info.hpp>
+
+#include <ngraph/pattern/op/wrap_type.hpp>
+
+NGRAPH_RTTI_DEFINITION(ngraph::pass::ConvertCTCGreedyDecoderV6ToV1, "ConvertCTCGreedyDecoderV6ToV1", 0);
+
+ngraph::pass::ConvertCTCGreedyDecoderV6ToV1::ConvertCTCGreedyDecoderV6ToV1() {
+    MATCHER_SCOPE(ConvertCTCGreedyDecoderV6ToV1);
+    auto decoder = pattern::wrap_type<opset6::CTCGreedyDecoderSeqLen>();
+
+    ngraph::matcher_pass_callback callback = [=](ngraph::pattern::Matcher& m) {
+        auto & pattern_value = m.get_pattern_value_map();
+        const auto & m_decoder = pattern_value.at(decoder);
+        auto decoder_v6 = std::dynamic_pointer_cast<ngraph::opset6::CTCGreedyDecoderSeqLen> (m_decoder.get_node_shared_ptr());
```

Comment:
You have only one node in pattern and it is root. So it's easier to use `m.get_match_root()`
```suggestion
        auto decoder = std::dynamic_pointer_cast<opset6::CTCGreedyDecoderSeqLen> (m.get_match_root());
```